### ISSUE TITLE
Mobie seems to have problems with ilastik as first image (image to get metadata from)

### ISSUE THREAD

**Issue by k-dominik** [mobie/mobie-io#166]
This line looks MoBiE has problems accepting ilastik h5 as "first" image... The following line looks suspicious - or is that fine? I tried with a tif as first image and h5 as "secondary", and this worked fine.

https://github.com/mobie/mobie-io/blob/a3f73218c513c5de47df81c23219f8a556651d9b/src/main/java/org/embl/mobie/io/imagedata/IlastikImageData.java#L108

**Comment by tischi** [comment]
I don't think this has anything to do with the issue, this is about opening one image file containing multiple "datasets", which for mobie typically means multiple channels, as we have a single channel image data model.

**Comment by tischi** [comment]
I just tried and this works (here the "first image" is an "ilastik image"):

```
	@Test
	public void test( )
	{
		new ImageJ().ui().showUI(); // initialise SciJava Services

		final OpenImageAndLabelsCommand command = new OpenImageAndLabelsCommand();
		command.image = new File( "src/test/resources/ilastik-2d/probabilities.h5" );
		command.labels = new File( "src/test/resources/ilastik-2d/labels.h5" );
		command.table = new File( "src/test/resources/ilastik-2d/table.csv" );
		command.run();
	}
```

So I am not sure what the issue is; I would need a more specifics....
